test/src/graphTest.cpp

```diff
@@ -220,6 +220,64 @@ TEST_F(Graph, FormatNodeResults)
 	}
 }
 
+TEST_F(Graph, UsePointsFromArray)
+{
+	rgl_node_t usePoints=nullptr, transform=nullptr, format=nullptr;
+
+	struct PointStruct
+	{
+		Field<XYZ_F32>::type xyz;
+		Field<PADDING_32>::type padding;
+		Field<IS_HIT_I32>::type isHit;
+		Field<INTENSITY_F32>::type intensity;
+	} pointStruct;
+
+	std::vector<rgl_field_t> pointFields = {
+		XYZ_F32,
+		PADDING_32,
+		IS_HIT_I32,
+		INTENSITY_F32
+	};
+
+	std::vector<PointStruct> inPoints;
+	inPoints.push_back(PointStruct { .xyz = {1, 2, 3}, .isHit = 1, .intensity = 100 } );
```

Comment:
My suggestion is to change the test to TEST_P and parametrize it with some corner cases.
0 elements, 1 element, 1241251 elems...

Reply:
But, we can leave it, and I will rewrite it with the next graph_test task.

Reply:
Good idea. Let's rewrite it in the new task.